modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -86,12 +86,13 @@ def new
   def create
     service_result = Storages::Storages::CreateService.new(user: current_user).call(permitted_storage_params)
     @object = service_result.result
```

Comment:
`@object` should only be assigned if the `service_result` is indeed successful. I suggest moving it to line 92 inside the truthy `if` branch, as `@object` is not further used inside this controller method.